linux_pipeline/Jenkinsfile_rhel_kernel_validation

```diff
@@ -173,3 +173,37 @@ stage ("Send Email Report"){
         }
     }
 }
+
+stage ("Trigger Hotfix"){
+    node ("jenkins-meta-slave") {
+        if (!sendMail) {
+            return
```

Comment:
What is the use of this?

Reply:
This helps to return from each stage when there is no new errata kernel detected